src/core/bpf/allow_bind/allow-bind.bpf.c

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: GPL */
+
+/* <linux/types.h> must precede <bpf/bpf_helpers.h> due to integer types
+ * in bpf helpers signatures.
+ */
+#include <linux/types.h>
+#include <bpf/bpf_endian.h>
+#include <bpf/bpf_helpers.h>
+#include <linux/bpf.h>
+#include <netinet/in.h>
+
+/* Binding is allowed only to ports specified in ports_v{4|6} map.
+ * Ports are in host order.
+ * max_entries is set from user space by bpf_map__resize helper.
+ */
+struct {
+        __uint(type, BPF_MAP_TYPE_HASH);
+        __type(key, __u16);
+        __type(value, __u8);
+        __uint(max_entries, 0);
```

Comment:
Is it possible to just remove this line?

Reply:
yes, libbpf doesn't care